src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -318,6 +319,12 @@ private void parse(String [] argv) {
 			properties = parseFile(DEFAULT_CONFIG_FILE, false);
 		}
 
+		if (options.has("env_config_prefix")) {
+			String prefix = ((String) options.valueOf("env_config_prefix")).toLowerCase();
+			System.getenv().entrySet().stream().filter(map -> map.getKey().toLowerCase().startsWith(prefix))
```

Comment:
What if the user makes a typo and have env like `maxwell_passwod`? Should we defend/filter against this scenario and/or throw a warning?

Reply:
Probably not, because if using config file you'll face the same problem. And the change uses the same mechanism as config file.